mlos_bench/mlos_bench/config/services/remote/azure/scripts/setup-rg/results-db/mysql-template.parameters.example.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "dbName": {
+            "value": "mlos-autotune-db"
+        },
+        "dbAdminUsername": {
+            "value": "mlos"
+        },
+        "dbAdminPassword": {
+            "value": "PLACEHOLDER"
+        },
+        "dbFirewallRules": {
+            "value": [
+                {
+                    "name": "PLACEHOLDER",
+                    "startIpAddress": "123.123.0.0",
```

Comment:
A better set of non-IPs to use would be something like the RFC1918 (private) addresses:

192.168.0.0
192.168.255.255

That way if they forget to change those they aren't accidentally opening things up to the public internet.

Reply:
That makes sense! Changed them to your suggestion.